samples/extensions/shader_debugprintf/shader_debugprintf.cpp

```diff
@@ -393,9 +415,37 @@ bool ShaderDebugPrintf::prepare(const vkb::ApplicationOptions &options)
 	return true;
 }
 
+// This sample overrides the per-sample layer framework to force activation of the validation layer
+const std::vector<const char *> ShaderDebugPrintf::get_validation_layers()
+{
+	// Note validation layer is already enabled for debug builds, so set the default list to empty
+	std::vector<const char *> validation_layers = {};
```

Comment:
Maybe instead:
`std::vector<const char *> validation_layers = ApiVulkanSample::get_validation_layers();`

Reply:
Yes, I can do this if you would like.  Can you tell me if `ApiVulkanSample::get_validation_layers()` will ever return non-null?  i.e. could there be a time in the future when it will contain other values?

Reply:
Are you asking me about the future???

It might be possible, yeah. Today, I can't imagine what might lead to such a situation.
But no matter what, I think, it's good practise to somehow explicitly handle the returned value from the base class when you override a function.

Reply:
Same as above,  will fix later tonight EDT.